portfolio/src/main/webapp/index.js

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {removeClassName, displayDefaultSection, displaySection, setToNone} from "./tab.js";
+import {showNextSlide, showSlide, showCurrentSlide, slideIndex} from "./slideshow.js";
+
+/** Allow all imported functions to be in the scope of "window" */
+window.onload = () => {
+  window.displaySection = displaySection;
+  window.setToNone = setToNone;
+  window.displayDefaultSection = displayDefaultSection;
+  window.removeClassName = removeClassName;
+  window.showNextSlide = showNextSlide;
+  window.showSlide = showSlide;
+  window.showCurrentSlide = showCurrentSlide;
+  window.slideIndex = slideIndex;
```

Comment:
Wrap the tab and slideshow functions in their own classes. You don't want to pollute the global namespace more than you have to, especially since any naming collisions will overwrite functionality leading to unintended behavior.